src/main/java/org/opentripplanner/standalone/config/buildconfig/NetexConfig.java

```diff
@@ -158,6 +158,29 @@ private static NetexFeedParameters.Builder mapFilePatternParameters(
           .docDefaultValue(dft.ferryIdsNotAllowedForBicycle())
           .asStringSet(base.ferryIdsNotAllowedForBicycle())
       )
+      .addRouteToCentroidStationIds(
+        config
+          .of("routeToCentroidStationIds")
```

Comment:
Do you want this list to be validated or should it stay silent if you put an unknown one in there?

Reply:
You mean validating that the station ids actually exist? I think that would be a nice feature, I'll see if i can find a good place to put such a check.

Reply:
On the other hand we have other features that allow you to specify IDs and often they simply ignore those that are not found.

Reply:
I added validation for this.